proto/zitadel/system.proto

```diff
@@ -763,18 +799,33 @@ message RemoveQuotaResponse {
 }
 
 message SetLimitsRequest {
-  string instance_id = 1 [(validate.rules).string = {min_len: 1, max_len: 200}];
+  string instance_id = 1 [(validate.rules).string = {len: 18}];
```

Comment:
I added this because I successfully created limits while testing with quoted IDs `"123..."`.
Because we don't query the instances, we can't validate the IDs, so I wanted to at least handle unintended wrong formats like in the mistake I made. I'll remove it for now, because if we want to handle this, we should probably handle it everywhere the same. And with enough great power to use the system API also comes great responsibility.

Reply:
done